src/lib/containers/entity_finder/SelectionPane.tsx

```diff
@@ -75,7 +75,8 @@ const EntityPathDisplay: React.FunctionComponent<{
 
   useEffect(() => {
     if (bundle?.path?.path) {
-      setEntityName(bundle.path.path[bundle.path.path.length - 1].name)
+      const header = bundle.path.path[bundle.path.path.length - 1]
+      setEntityName(header.name ?? header.id)
```

Comment:
If lacking READ access, it's likely that `name` will be undefined (pending the outcome of [PLFM-7654](https://sagebionetworks.jira.com/browse/PLFM-7654))